src/component/history/EventLog/index.tsx

```diff
@@ -0,0 +1,27 @@
+import EventLog from './EventLog';
+import { useEventSettings } from "../../../hooks/useEventSettings";
+import { useLocationSettings } from "../../../hooks/useLocationSettings";
+
+interface IEventLogContainerProps {
+    title: string;
+    history: unknown[];
+    displayInline?: boolean;
+}
+
+const EventLogContainer = (props: IEventLogContainerProps) => {
+    const [locationSettings] = useLocationSettings();
```

Comment:
Is there a significant benefit here of using an array instead of an object? I know we can't keep it 100% consistent across all hooks, but if we can land a pattern for our own hooks then it will help reducing the cognitive load (types will also help). I think almost all of our hooks currently return objects.

Reply:
Good point. I was mimicking `useState`, but the object-form would enforce consistent names. Will fix.

Reply:
I don't blame you! I think it's sensible in a lot of situations, but I've found that the array form sometimes can be a bit limiting especially when returning more than two values - or when you just need the second item in the array, resulting in:
```
[, setSomething] = useSomething();
``` 

With setState it's not that much of an issue since you'll always need the piece of state and setter (otherwise you wouldn't need state)